packages/react-magma-dom/src/components/Pagination/Pagination.tsx

```diff
@@ -65,6 +66,11 @@ export interface BasePaginationProps
    * @default false
    */
   showLastButton?: boolean;
+  /**
+   * If true, shows the Pagination pages in a dropdown style.
+   * @default false
+   */
+  simple?: boolean;
```

Comment:
**Question for consideration:**
In other components when we have different types/versions, we use a `type` prop, and we set a default (example: Button, LoadingIndicator, List, Input). Should we match that for consistency, or is it better to have a boolean prop?

Reply:
Converted to an enum for consistency!